### PR TITLE
[4.0] Improve Text on Modal: Batch process language labels

### PR DESCRIPTION
**Pull request by pe7er**
IMHO some of the text in the **Batch process the selected articles** modal is too long:
- "To Move or Copy your selection please select a Category." 
- "Do you want to move the items or make a copy of them?" 

This PR rewrites that text to:
- "Select a Category for your Selection"
- "Copy or Move the items?"

### Summary of Changes
On the current modal some of the text seems a bit too long:
batch2

### Testing Instructions
Content > Articles > select some articles > click on Batch
Select a category so that the Copy / Move question is shown.

### Expected result
This PR makes two of the labels shorter so that IMHO it looks nicer,
but also easier to read and understand:

batch3

### Suggestion (NOT in this patch)
If this PR gets accepted then I suggest to move the radio buttons of Copy / Move to the left

batch4-suggestion

by removing the left margin in administrator/templates/atum/css/template.css line 470:
```CSS
.control-group .controls {
    margin-left: 220px;
}
```

### CODE REVIEW

**Comment by pe7er** [comment]
I can create a similar PR for:
-  "Batch process the selected menu items" (Menu > Batch)
-  "Batch process the selected fields." (Content > Fields > Batch)

**Comment by brianteeman** [comment]
The length of the text should not be a consideration as it will always be different lengths in other languages

**Comment by pe7er** [comment]
> The length of the text should not be a consideration as it will always be different lengths in other languages

I know. Could you please look at the suggested improvement itself (besides the length) ?

**Comment by infograf768** [comment]
Although I do like the change for
`JLIB_HTML_BATCH_MOVE_QUESTION="Copy or Move the items?"`
I am not sure for the other one as it gives the feeling that one HAS to select a category.

**Comment by Quy** [comment]
Is it necessary to include `for your Selection` and `the items`? This is not done with the other labels. 
Plus, it is implied in the heading `Batch process the selected articles`.

Maybe remove the second label and replace the first label with `Move or Copy to Category`.

**Comment by opn365** [comment]
I have tested this item :white_check_mark: successfully on 734666a5a05c5bb08b459bc7753b0aa6edc8a8ca<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24817">issues.joomla.org/tracker/joomla-cms/24817</a>.</sub>

**Comment by jduerscheid** [comment]
I have tested this item :white_check_mark: successfully on 734666a5a05c5bb08b459bc7753b0aa6edc8a8ca<br /><br />It works<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24817">issues.joomla.org/tracker/joomla-cms/24817</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24817">issues.joomla.org/tracker/joomla-cms/24817</a>.</sub>

File: administrator/language/en-GB/en-GB.lib_joomla.ini [hunk 1]
```diff
@@ -361,9 +361,9 @@ JLIB_HTML_BATCH_FLIPORDERING_LABEL="Reverse the ordering of all articles in the
 JLIB_HTML_BATCH_LANGUAGE_LABEL="Set Language"
 JLIB_HTML_BATCH_LANGUAGE_LABEL_DESC="Not making a selection will keep the original language when processing."
 JLIB_HTML_BATCH_LANGUAGE_NOCHANGE="- Keep original Language -"
-JLIB_HTML_BATCH_MENU_LABEL="To Move or Copy your selection please select a Category."
+JLIB_HTML_BATCH_MENU_LABEL="Select a Category for your Selection"
 JLIB_HTML_BATCH_MOVE="Move"
-JLIB_HTML_BATCH_MOVE_QUESTION="Do you want to move the items or make a copy of them?"
+JLIB_HTML_BATCH_MOVE_QUESTION="Copy or Move the items?"
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.lib_joomla.ini:366]:
```suggestion
JLIB_HTML_BATCH_MOVE_QUESTION="Move or Copy the items?"
```
To be consistent with the next language string.

Comment (wilsonge) [reply administrator/language/en-GB/en-GB.lib_joomla.ini:366]: Given the copy option shows before the move option in the UI probably the correct thing is to change the string below too (and also keeps it alphabetical)

**Comment by wilsonge** [comment]
I agree with JM here to an extent. Whilst I think it's clear you don't have to select a category (as the default is "Don't move or copy". Which is the same for all other options that modify data (otherwise you can argue they are all also issues), I would imagine the default string in the value field being the only way of understanding you can move or copy items could be confusing to people unfamiliar to batch (I hesitate to say new users because batch isn't exactly a brand new user feature)

I'd be interested in hearing more opinions here before merging (clearly no need to more testers as the patch itself is self-explanatory)

**Comment by Quy** [comment]
Please continue discussion/testing in PR #28447.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24817">issues.joomla.org/tracker/joomla-cms/24817</a>.</sub>

**Pull request closed without merging**